packages/settings-library/src/settings_library/node_ports.py

```diff
@@ -13,6 +13,7 @@
 class StorageAuthSettings(StorageSettings):
     STORAGE_USERNAME: str | None
     STORAGE_PASSWORD: SecretStr | None
+    STORAGE_SCHEME: str = "http"
```

Comment:
ok can yo uplease check the other ENV and use the same wording? it is already complicated enough.
Either go with:
- STORAGE_URL
- or, STORAGE_SECURE
- or change all of them to SCHEME

Reply:
As discussed, let's adapt and go for `{prefix_}_SECURE` by modifying `MixinServiceSettings`